_posts/2018-04-09-synching-sia.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Syncing Sia in 73 minutes
+layout: post
+date: "2018-04-09"
+description: "An experiment to see how fast I can perform the initial Sia blockchain sync"
+tags:
+- sia
+- blockchain
+- benchmark
+permalink: "/sia-sync-73-min/"
+---
+
+*This is a guest post by Sia community contributor [Thomas Bennett](https://github.com/tbenz9).*
+
+I can sync Sia from a block height of 0 to 149000 in 73 minutes.  Here’s how I do it.
+
+## Zero subscribers
+
+As mentioned in my previous article [Benchmarking Sia’s Blockchain Sync](https://blog.spaceduck.io/sia-blockchain-sync/) Sia spends an average of 55 minutes or 19.1% of its time notifying subscribers.  I submitted the pull request [#2809](https://github.com/NebulousLabs/Sia/pull/2809) so Sia only notifies subscribers if there are any.  By running Sia with only the `consensus` and `gateway` modules loaded there will be zero subscribers.
+
+My `siad` command with minimal modules loaded.
+
+```shell
+./siad -M cg -d /root/Sia/
```

Comment:
I prefer to always use long flag names in blog post so it's clear to the reader what the flag is, so I think `--modules cg` is better.

Suggest dropping the `-d /root/Sia` part because it's not related to eliminating subscribers.